### PR TITLE
Serves a list of the current users with express

### PR DESCRIPTION
**Pull request by jeversmann**
Current API serving code; the index page is temporary, and should be replaced with a redirect to the `gh-pages` site in the future.

Most of this code is default Express bootstrapping.

### CODE REVIEW

**Comment by JDong820** [comment]
We can’t automatically merge this pull request.

File: app.js [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+    var err = new Error('Not Found');
+    err.status = 404;
+    next(err);
+});
+
+// error handlers
+
+// development error handler
+// will print stacktrace
+if (app.get('env') === 'development') {
```
Comment (JDong820) [inline app.js]: how does this work? can you hold my hand?

Comment (jeversmann) [inline app.js]: This has to do with configuration settings when node is started. It defaults to development mode, and can be changed to production mode by setting shell environment variables before running node: `export NODE_ENV=prodction`.

**Comment by jeversmann** [comment]
Rebased for mergability and 10% more happiness.

File: routes/current.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
+var express = require('express');
+var router = express.Router();
+
+// Serves every current record
+router.route('/')
+  .get(function(req, res) {
+    req.db.get('current').find({}, {},
```
Comment (JDong820) [inline routes/current.js]: Is this the right way to `select * from *`

**Comment by JDong820** [comment]
Seems sensible. How should we start documenting the API, if only for ourselves?

**Pull request merged**